src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -187,12 +215,16 @@ bool mongoConnect(const char*         host,
     //
     // WriteConcern
     //
-    int writeConcernCheck;
+    mongo::WriteConcern writeConcernCheck;
```

Comment:
@kzangeli maybe you find interesting this part... it seems that the writeConcern has changed in legacy, so I needed to adapt your code a little bit. Check if it is ok, please.